src/main/webapp/resources/js/components/project/ProjectSelect.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Select, SelectProps, Tag, Typography } from "antd";
+
+export type Project = { identifier: number; name: string };
```

Comment:
type `Project` is already defined in `types/irida/index.d.ts`.  If you don't want all the attributes, you could do something like:

```typescript
const SelectProject = Pick<Project, "id", "name">
```

Reply:
I noticed that ProjectMinimal is the same. Could I use that?

Reply:
You should in fact use that :)

Reply:
Please see 76ba8665596da5cb030453f0907409efa334b225.